tensorbay/opendataset/BDD100K/loader.py

```diff
@@ -279,6 +360,82 @@ def _merge_label(source_label_contents: List[List[Dict[str, Any]]]) -> Dict[str,
     return label_contents
 
 
+def _load_masks(
```

Comment:
do we need to wrap this function? this function modifies the input argument which is not recommended, and it just be called once.